MapboxSearch/base/src/main/java/com/mapbox/search/base/result/SearchResultFactory.kt

```diff
@@ -77,15 +78,20 @@ class SearchResultFactory(private val recordResolver: IndexableRecordResolver) {
                     return AsyncOperationTaskImpl.COMPLETED
                 }
             }
-            CoreApiType.SBS, CoreApiType.AUTOFILL -> {
-                if (searchResult.action == null && searchResult.type != BaseRawResultType.USER_RECORD) {
-                    failDebug { "Can't create search suggestion from. ${debugInfo()}" }
-                    callback(Result.failure(Exception("Can't create search suggestion from $searchResult")))
-                    return AsyncOperationTaskImpl.COMPLETED
+            CoreApiType.SEARCH_BOX, CoreApiType.AUTOFILL -> {
+                if (searchResult.action == null) {
+                    if (searchResult.type == BaseRawResultType.QUERY) {
+                        callback(Result.failure(InternalIgnorableException("Skipping query suggestion without action")))
+                        return AsyncOperationTaskImpl.COMPLETED
+                    } else if (searchResult.type != BaseRawResultType.USER_RECORD) {
+                        failDebug { "Can't create search suggestion from. ${debugInfo()}" }
+                        callback(Result.failure(Exception("Can't create search suggestion from $searchResult")))
+                        return AsyncOperationTaskImpl.COMPLETED
+                    }
                 }
             }
-            CoreApiType.SEARCH_BOX -> {
-                callback(Result.failure(Exception("Search Box is not supported yet")))
+            CoreApiType.SBS -> {
```

Comment:
check

Reply:
Wait, this one is legit. We are moving from SBS to SearchBox, so here we report an error for SBS now.